source/common/common/utility.h

```diff
@@ -282,6 +282,21 @@ class StringUtil {
                                                    absl::string_view delimiters,
                                                    bool keep_empty_string = false);
 
+  /**
+   * Remove tokens from a delimiter-separated string view.
+   * @param source supplies the delimiter-separated string view.
+   * @param multi-delimiter supplies chars used to split the delimiter-separated string view.
+   * @param tokens_to_remove supplies a set of tokens which should not appear in the result.
+   * @param joiner contains a string used between tokens in the result.
+   * @param trim_tokens trims the the tokens before comparing to 'tokens_to_remove'.
+   * @param ignore_case compares the tokens in a case-insensitive way to tokens_to_remove'.
```

Comment:
It seems the implementation is not exactly the same.
`tokens_to_remove` is not applying `toLower`. 

Add an extra comment to address it?